### DISCUSSION TITLE
Option to recall memories and daily notes

### DISCUSSION THREAD

**Discussion by sonct988** [Kageetai/obsidian-plugin-journal-review#122] [Ideas]
I'm using journal review to recall memories (similar to this day last year on Facebook), and along with it I still have daily notes. Now when I want to separate them from each other, just to recall memories, not related to daily notes, I have difficulty finding the option to configure folders so that the plugin only reads and recalls memories. date in there. Is it possible to add an option to do that?? help me

**Comment by Kageetai** [comment]
Hey there,
sorry for the late answer, and thanks for the idea.
For the time being, this plugin is only meant to work with daily notes, not any other kind of notes.
Can you tell me more how you have these memories configured differently than the daily notes? 
How would you imagine this to work, then? Should the plugin look into other folders than the default folder for daily notes? Or also treat them differently in other ways?

**Comment by sonct988** [comment]
I think we should add the option to set the path for the active plugin (file location).

**Comment by sonct988** [comment]
It is also possible to create 2 different display tabs, 2 different paths. Memories is an extension option that can be enabled/disabled

![CleanShot 2024-08-21 at 11 15 00@2x](https://github.com/user-attachments/assets/5692f08f-e6f2-48f2-bd30-a9626886f32f)

**Comment by Kageetai** [comment]
I have to look into this, but it might require quite some reworking of the code unfortunately. Can't promise that any time soon I think, but I have it on my list. Thanks

**Reply by sonct988** [reply]
Thanks for your interest in this proposal.

**Comment by ZorkC64-128** [comment]
Hi, I love this plugin. I love reviewing 'On this day' as I have journaled my entire adult life. I was wondering if you could put an option for the Journal Review to only show text below a heading or subheading. My journal entries have multiple sections, but the section that has my personal thoughts is further down into the note. Maybe it already does this as I'm not exactly sure what the regex option thing does. Anyway, keep up the good work. Thank you for your contributions to the Obsidian community.

**Reply by Kageetai** [reply]
Hey there, that's a nice idea indeed. I think I had that somewhere in my notes as well :)
But no, the regex could not achieve that, I realise now that the description is rather unclear, as I had only added that on specific request by someone else. Should clarify what it means and does. :)

Would you mean creating another discussion item instead of here, as it's a slightly different thing, than what has been discussed above. Just makes it easier to follow up on and discuss. Thanks :)
